e2e/Pages/ChatComponent.ts

```diff
@@ -0,0 +1,65 @@
+import { expect } from '@playwright/test';
+import { Locator, Page } from 'playwright';
+
+import EnglishTranslations from '../../interfaces/IBF-dashboard/src/assets/i18n/en.json';
+import DashboardPage from './DashboardPage';
+
+const chatDialogueContentWelcomeNoTrigger =
+  EnglishTranslations['chat-component'].floods['no-event-no-trigger'].welcome;
+const chatDialogueWarnLabel =
+  EnglishTranslations['chat-component'].common['warn-label'].message;
+
+class ChatComponent extends DashboardPage {
+  readonly page: Page;
+  readonly chatDialogue: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.page = page;
+    this.chatDialogue = this.page.getByTestId('dialogue-turn-content');
+  }
+
+  async chatColumnIsVisibleForNoTriggerState({
+    name,
+    surname,
```

Comment:
We use `lastName` in the [dashboard](https://github.com/rodekruis/IBF-system/blob/5bb38d45f41e84c4e890b04315f7593b8bdb8be7/interfaces/IBF-dashboard/src/app/models/user/user.model.ts) and [API](https://github.com/rodekruis/IBF-system/blob/5bb38d45f41e84c4e890b04315f7593b8bdb8be7/services/API-service/src/api/user/user.entity.ts#L43). Let's keep the naming convention consistent. We also have a `middleName` in case this makes a difference to the test.

Reply:
Changed to "firstName" and "lastName"